net.c

```diff
@@ -2339,6 +2306,52 @@ int _ifcfg_write(char *device, ifcfg_t *ifcfg)
   return ok;
 }
 
+// update the sysconfig file *filename* with the data in *slist*
+//
+// slist is mutated by appending "=" to the keys
```

Comment:
it would at least deserve mentioning that the procedure requires usage of a writable temporary file